docs/index.md

```diff
@@ -29,7 +28,7 @@ terraform {
   required_providers {
     vcfa = {
       source  = "vmware/vcfa"
-      version = "= 0.1.0"
+      version = "= 1.0.0"
```

Comment:
Simplify for ongoing maintenance.

```suggestion
      version = "= x.y.z"
```

Reply:
Good point, changed

Reply:
Stop. No. Our goal is easy copy/paste-ability of samples by users. This doesn't cost anything to update with each release, which we have been doing with no issues.

Reply:
I have updated to `"~> 1.0.0"` so it picks possible patches.
Also I think I can improve `TestProviderVersion` so it checks this one

Reply:
Good idea!

Reply:
> Stop. No. Our goal is easy copy/paste-ability of samples by users. This doesn't cost anything to update with each release, which we have been doing with no issues.
> 
> Reminder: user-experience and human-readability is priority in our docs.

For reference, we do this in the other providers and the Registry will provide the block for copy/paste based on the version in scope.